### PR TITLE
Todd

### PR DESCRIPTION
**Pull request by w0hrk**
Hey guys, I've added Todd's code and tweaked a couple things:

1. There's no more index.html (was: game.html). This gets generated by the build when you do either `npm start` during development or `npm run build` to build the zip file.
2. Since the index.html is boilerplate (it's basically just an empty html doc), the code now creates the canvas element instead of getting it. Minor tweak.
3. It also just calls main() at the end of the source module to start things.

So basically everything is self-contained in the TS/JS.

Also, I did another comparison between (a) building with `tsc` and `uglifyjs` and (b) building with `webpack`, and the webpack build was ~100 bytes larger. If that becomes an issue we can easily switch back to (a).

The README is updated.

### CODE REVIEW

**Comment by w0hrk** [comment]
Forgot to add: the zip file is 1520 bytes right now. So that's pretty good it seems like. Kinda crazy, but the original Atari games were 4K. So this js13k thing is very generous. :)
